### PR TITLE
chore(tooling): add stylelint to start linting scss and initial rule set

### PR DESCRIPTION
**Pull request by emoralesb05**
the rule set is just a draft, the rules are subject to change.

### What's included?

- `npm run stylelint` to do initial linting for scss files
- Rename `npm run lint` to `npm run tslint`

#### Test Steps

- [ ]  Install VS Code extension `stylelint`
- [ ] `npm run stylelint` and see all the errors (we will fix in consecutive PR's)
- [ ] `npm run tslint` and see it work

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:release` still works.

closes https://github.com/Teradata/covalent/issues/278

### CODE REVIEW

**Comment by jotatoledo** [comment]
You could consider to add the markdown/javascript linters as well

**Comment by emoralesb05** [comment]
@jotatoledo any suggestions on a linter for html?? would be nice we can add custom rules for angular attrs also

**Comment by jotatoledo** [comment]
Havent found any yet ;(

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   ],
   "scripts": {
     "webdriver-update": "bash ./node_modules/.bin/webdriver-manager update",
-    "lint": "tslint --format codeFrame -c ./tslint.json \"./src/**/*.ts\" -e \"./src/**/typings.d.ts\" -e \"./src/environments/**\"",
+    "tslint": "./node_modules/.bin/tslint --format codeFrame -c ./tslint.json \"./src/**/*.ts\" -e \"./src/**/typings.d.ts\" -e \"./src/environments/**\"",
+    "stylelint": "./node_modules/.bin/stylelint 'src/**/*.scss' '!**/_layout.scss' '!**/markdown.component.scss' --config .stylelintrc.json --syntax scss",
```
Comment (jeremysmartt) [inline package.json]: should we also have a target that does both tslint and stylelint in command?

Comment (emoralesb05) [reply package.json]: I was debating that also, i think we can use `npm run lint` to run both

Comment (jeremysmartt) [reply package.json]: 👍

**Pull request merged**